iiasa/ixmp
# Pin fastjsonschema for all pythons

For now, this is just an attempt to fix today's test failure. fastjsonschema changing from v2.20.0 to 2.21.0 was the only difference I could find to yesterday's successful run, though I'm not convinced this is the culprit. 

It seems like pinning fastjsonschema actually works, but I still don't understand why, unfortunately, as it doesn't show up in the traceback.

## How to review


- Read the diff and note that the CI checks all pass.
- Run a specific code snippet or command and check the output.
- View the preview build of the documentation and look at a certain page.
- Ensure that changes/additions are self-documenting, i.e. that another
  developer (someone like the reviewer) will be able to understand what the code
  does in the future.

## PR checklist
